tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py27,py35
+
+[testenv]
+deps =
+    -rrequirements-dev.txt
+
+commands=
+   py.test --cov-report= --cov=napalm_eos test/
+   pylama napalm_eos test
```

Comment:
Yeah, please, put `pylama .` back and add this to `setup.cfg` in the `pylama` section; `skip = .tox/*`. That should do.